cmd/vic-machine/create/create.go

```diff
@@ -543,6 +550,10 @@ func (c *Create) processParams() error {
 		return err
 	}
 
+	if err := c.processDefaultVolumeStore(); err != nil {
```

Comment:
Currently it will not, neither will several other functions called here. I decided to put it in this context because the surrounding process functions also have this pattern. I will look at changing it :)